frontend/src/components/Item/Metadata/ImageSearch.vue

```diff
@@ -3,59 +3,66 @@
     :model-value="dialog"
     :fullscreen="$vuetify.display.mobile"
     content-class="image-search-dialog-content"
-    width="60%"
-    @click:outside="$emit('update:dialog', false)">
+    @update:model-value="(value) => emit('update:dialog', value)">
     <v-card height="100%" class="image-search-card">
-      <v-card-title>{{ $t('search.name') }}</v-card-title>
+      <v-card-title>{{ t('search.name') }}</v-card-title>
       <v-divider />
       <v-row align="center" class="mx-16 my-4">
         <v-select
           v-model="source"
           class="mx-4"
           :items="sources"
           :disabled="loading"
-          :label="$t('metadata.source')"
+          :label="t('metadata.source')"
+          :placeholder="t('metadata.sourceAll')"
+          persistent-placeholder
           variant="outlined"
-          hide-details />
+          hide-details
+          clearable />
         <v-select
           v-model="type"
           class="mx-4"
           :items="types"
+          item-title="text"
+          item-value="value"
```

Comment:
Vuetify 3's default for the label is now `title` rather than `text` (https://vuetifyjs.com/en/api/v-select/). Leaving them blank results in
<img width="791" alt="image" src="https://user-images.githubusercontent.com/8809837/225546928-acb8888f-66a1-439d-9d34-99adead4b241.png">
:(

I had to make this change in a few uses of v-select to get them working again. This is something that could actually be enforced with typings on Vuetify's end to make sure that the default keys, or passed in keys, exist in the options object which are passed in and give a type error if not. Sadly I've learned not to expect much from Vuetify's types though.

I think there's 2 options here:
1. Update all places we pass in an options list to use the new default keys
2. Specify the text/value keys on the use of the component.

Because of the bad Vuetify typings, I think (2) is a bit more durable in the event of another Vuetify upgrade (however unlikely that may be) and my preference would be to leave this as is.

Reply:
No problem, just leave this as is! I just wanted to make you aware of the defaults (at least according to their docs), but it's a super nitpicky thing that I don't care about!